setup.py

```diff
@@ -20,8 +23,12 @@ def get_baked_files():
     """
     data_files = []
 
+    completion_dir = "/etc/bash_completion.d"
+    if platform.system() == "Windows":
+        completion_dir = "C:\Program Files\Git\mingw64\share\git\completion"
+
```

Comment:
Let me digging into this. I think Windows's shell environment can be complicated and we can't assume everybody uses the mingw64 inside Git bundle... If it's too complicated we might want to not implement the auto completion on windows.